tests/AvroConvertTests/TestClasses.cs

```diff
@@ -287,4 +287,14 @@ public record TestRecord
     {
         public string Name { get; set; }
     }
+
+#nullable enable
+    [Equals(DoNotAddEqualityOperators = true)]
+    public class ClassWithNullableMembers
+    {
+        public string? NullableStringProperty { get; set; }
```

Comment:
I believe, after your change, it could be simply "string" type?
If so, please change string? to string

Reply:
No this is correct. Since `string` is a reference type and marked explicitly as nullable, we want the schema to declare this as both `string` and `null`. Even though technically it's a reference type and therefore nullable, I didn't want to go down the route of marking it nullable in the schema since this would be quite a big breaking change.

With the above, the outputted schema should be:

```json
{"name": "NullableStringProperty", "type": ["string", "null"]}
```

If this was not-nullable, ie `public string Property { get; set; }`, I'd expect the schema to be:

```json
{"name": "Property", "type": "string"}
```

Does that make sense?

Reply:
Sure,
Thanks for the PR

Reply:
Thanks!